packages/server/email/getMailManager.ts

```diff
@@ -9,7 +9,8 @@ let mailManager: MailManager
 const managers = {
   mailgun: MailManagerMailgun,
   google: MailManagerGoogle,
-  smtp: MailManagerSMTP
+  smtp: MailManagerSMTP,
+  debug: MailManagerDebug
```

Comment:
Previously, tests were using the real MailGun instance. Now the `test` env uses the debug mailer.